projects/ngrx.io/content/guide/migration/v7.md

```diff
@@ -0,0 +1,139 @@
+# Migration guide v7
+
+## @ngrx/store
+
+### `select`
+
+In NgRx 6.1 the `select` static function was marked as deprecated in favor of the `select` RxJS operator, in NgRx v7 this static function was dropped.
+
+BEFORE:
+
+```ts
+import { Store } from '@ngrx/store');
+import { selectUser } from '../reducers');
+
+@Component({...})
+export class AppComponent implements OnInit {
+  user$: Observable<User>;
+
+  constructor(private store: Store<AppState>) { }
+
+  ngOnInit() {
+    this.user$ = this.store.select(selectUser);
+  }
+}
+```
+
+AFTER:
+
+```ts
+import { Store, select } from '@ngrx/store'); // import `select`
+import { selectUser } from '../reducers');
+
+@Component({...})
+export class AppComponent implements OnInit {
+  user$: Observable<User>;
+
+  constructor(private store: Store<AppState>) { }
+
+  ngOnInit() {
+    this.user$ = this.store.pipe(select(selectUser)); // use the pipeable select operator
+  }
+}
+```
+
+### `@ngrx/store/update-reducers` action
+
+When multiple feature reducers are registered, a single action is dispatched instead of an action for each added feature reducer.
+
+BEFORE:
+
+When adding/removing one feature:
+
+```ts
+{type: '@ngrx/store/update-reducers', feature: 'feat1'}
+```
+
+When adding/removing multiple features:
+
+```ts
+{type: '@ngrx/store/update-reducers', feature: 'feat1'}
+{type: '@ngrx/store/update-reducers', feature: 'feat2'}
+```
+
+AFTER:
+
+When adding/removing one feature:
+
+```ts
+{type: '@ngrx/store/update-reducers', features: ['feat1']}
+```
+
+When adding/removing multiple features:
+
+```ts
+{type: '@ngrx/store/update-reducers', features: ['feat1', 'feat2']}
+```
+
+## @ngrx/effects
+
+### `ofType`
+
+In NgRx 6.1 the `ofType` static function was marked as deprecated in favor of the `ofType` RxJS operator, in NgRx v7 this static function was dropped.
+
+BEFORE:
+
+```ts
+import { Effect, Actions } from '@ngrx/effects';
+
+@Injectable()
+export class MyEffects {
+  @Effect()
+  someEffect$: Observable<Action> = this.actions$
+    .ofType(UserActions.LOGIN)
+    .pipe(map(() => new AnotherAction()));
+
+  constructor(private actions$: Actions) {}
+}
+```
+
+AFTER:
+
+```ts
+import { Effect, Actions, ofType } from '@ngrx/effects'; // import ofType operator
+
+@Injectable()
+export class MyEffects {
+  @Effect()
+  someEffect$: Observable<Action> = this.actions$.pipe(
+    ofType(UserActions.LOGIN), // use the pipeable ofType operator
+    map(() => new AnotherAction())
+  );
+
+  constructor(private actions$: Actions) {}
+}
+```
+
+## @ngrx/store-router
```

Comment:
`## @ngrx/store-router`

This should be `@ngrx/router-store` instead of `@ngrx/store-router`, shouldn't it?

Reply:
Yea, you're right 😅